README.md

```diff
@@ -10,8 +10,8 @@ Only showing rspec tests for now:
 Cross-browser testing done with <a href="https://www.browserstack.com/"><img src="./Browserstack-logo@2x.png?raw=true" height="36" ></a>
 
 ## Setup
-* Use Ruby 2.3.7 (Set the version using [RVM](https://rvm.io/rvm/install) or [rbenv](https://github.com/rbenv/rbenv))
-* Install [Bundler](https://github.com/bundler/bundler) for this version of Ruby `gem install bundler -v 1.16.2`
+* Use Ruby 2.6.3 (Set the version using [RVM](https://rvm.io/rvm/install) or [rbenv](https://github.com/rbenv/rbenv))
```

Comment:
I updated Ruby version and bundler version here, is there anywhere else in the README?

Reply:
@akegan

Reply:
oh whoops. i hadn't seen that update, looks good then!